tests/osgi/functional/pom.xml

```diff
@@ -424,6 +430,7 @@
                                 <exclude>org/glassfish/jersey/osgi/test/basic/BasicOsgiIntegrationTest.java</exclude>
                                 <exclude>org/glassfish/jersey/osgi/test/basic/BeanValidationTest.java</exclude>
                                 <exclude>org/glassfish/jersey/osgi/test/basic/JaxRsRiBundleTest.java</exclude>
+                                <exclude>org/glassfish/jersey/osgi/test/basic/JsonJackson1Test.java</exclude>
```

Comment:
JsonJackson1Test is no longer in Jersey 3.x

Reply:
fixed